src/main/resources/lib/credentials/select.properties

```
@@ -24,4 +24,9 @@
 invalidExpression=A valid parameter expression consists of the parameter name enclosed \
   within <code>$'{'</code> and <code>'}'</code>
 fixedSelection=Specific credentials
-paramExpression=Parameter expression
\ No newline at end of file
+paramExpression=Parameter expression
+includeUserCredentials=List user credentials
+includeUserCredentialsHelp=Whether or not to include user credentials in the available \
```

```suggestion
includeUserCredentialsHelp=Whether to include user credentials in the available \
```
Very minor suggestion, but the "or not" is duplicative and takes up screen space.